proto/src/main/proto/Rss.proto

```diff
@@ -463,6 +464,12 @@ message AccessClusterResponse {
   string uuid = 3;
 }
 
+message FetchClientConfRequest {
+  string user = 1;
+  // for the potential extension for customize delegation shuffle manager
```

Comment:
Could you elaborate more?

Reply:
I will put more internal job properties in internal DelegationShuffleManager, and then extend the internal ClientConfStrategy to apply specific rss conf to job.

Reply:
Could you give more comments for this pull request?

Reply:
done